hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -338,7 +351,9 @@ private void validateMaybeNewFeeScheduleKey(
             @NonNull final Topic topic) {
         if (op.hasFeeScheduleKey()) {
             validateTrue(topic.hasFeeScheduleKey(), FEE_SCHEDULE_KEY_CANNOT_BE_UPDATED);
-            attributeValidator.validateKey(op.feeScheduleKey(), INVALID_CUSTOM_FEE_SCHEDULE_KEY);
+            if (!KeyOneOfType.UNSET.equals(op.feeScheduleKey().key().kind())) {
```

Comment:
Not needed,validateKey() takes care of it.

Reply:
validateKey will throw an error if it's UNSET. We want to call validateKey only if it's an actual Key.